Donation-platform/src/main/java/com/google/servlets/TagScoreUpdateServlet.java

```diff
@@ -56,17 +58,25 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
     public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
         DatastoreService ds = DatastoreServiceFactory.getDatastoreService();
+        DbCalls dbCalls = new DbCalls(ds);
```

Comment:
This should be relavant to Toni.

DBCalls should be singleton class and should be initialized at program startup. Avoid creating multiple instances of the DB handle. 
Use singleton class for DbCalls: https://www.geeksforgeeks.org/singleton-class-java/
Use constructor to initialize ds variable